chocolatey-diff/Public/Get-ChocolateyPackageDiff.ps1

```diff
@@ -107,6 +107,31 @@
     Expand-Archive -Path $oldFileName -DestinationPath $oldExtractPath -Force
     Expand-Archive -Path $newFileName -DestinationPath $newExtractPath -Force
 
+    foreach ($path in @($oldExtractPath; $newExtractPath)) {
+        # Only because of unit tests
+        if (!(Test-Path $path)) { continue; }
```

Comment:
Why is this there? What is the use of this?
I think you'll need to create Mocks to get around this.

Reply:
I don't remember right now, I have had this code in a stash for quite some time.
I remember that the unit tests were failing until I added this statement, but the unit tests have been updated since the initial stash creation so it may not be needed any longer.

The comment was also more a reminder to myself to figure out why the unit test was failing.

Reply:
The Pester tests mock the folder and should not actually go out on the internet to download an actual package. I guess it's due to that.

You'll do need to make actual Pester tests.